src/fleet/azext_fleet/custom.py

```diff
@@ -101,6 +114,7 @@ def update_fleet(cmd,
                  name,
                  enable_managed_identity=None,
                  assign_identity=None,
+                 no_wait=False,
```

Comment:
good catch, but also need to update

g.custom_command("update", "update_fleet", supports_no_wait=True)

i.e. adding supports_no_wait there too.

Reply:
Likewise, good catch.